polls/templates/polls/experiment-type-annotation.html

```diff
@@ -0,0 +1,28 @@
+{% extends 'base.html' %}
+{% block body-block %}
+<div class="mt-5 mb-5 text-center">
+    <h2>Create Experiment-type Annotation</h2>
+    <p></p>
+</div>
+<form hx-post="{% url 'experiment-type-annotation' %}" hx-swap="innerHTML"
+    hx-target="#body" hx-boost="true" hx-headers='{"X-CSRFToken": "{{ csrf_token }}"}'>
+    
+    <div class="mb-3">
+        <label for="annotation" class="form-label">Input annotation pattern:</label>
+        <input type="text" id="annotation" class="form-control" name="annotation" placeholder="Input annotation pattern, E.G: AAB/ABA/BAB...">
```

Comment:
I think the in the placeholder we should have "XYX" or "XXY" instead of "AAB" because annotations are saved like former in our db.

Reply:
I'll resolve this now.